### ISSUE TITLE
model1

### ISSUE THREAD

**Issue by sugapriya958** [sugapriya958/sugapriya#1]
<html>
<body>
<div class="adn ads" data-message-id="#msg-f:1855736587270934371" data-legacy-message-id="19c0e69810facf63" style="border-left: none; padding: 0px; display: flex; color: rgb(34, 34, 34); font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-size: medium; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="gs" style="margin: 0px; min-width: 0px; padding: 0px 0px 20px; width: auto;"><div class=""><div class="ii gt" jslog="20277; u014N:xr6bB; 1:WyIjdGhyZWFkLWY6MTg1NTczNjU4NzI3MDkzNDM3MSJd; 4:WyIjbXNnLWY6MTg1NTczNjU4NzI3MDkzNDM3MSIsbnVsbCxudWxsLG51bGwsMSwwLFsxLDAsMF0sOTksODMyLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCwxLG51bGwsbnVsbCxbMF0sbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsMCwwLG51bGwsbnVsbCxudWxsLG51bGwsWzEsMF1d" id=":lk" style="direction: ltr; margin: 8px 0px 0px; padding: 0px; position: relative; font-size: 0.875rem; overflow-x: hidden;"><div class="a3s aiL" id=":lj" style="direction: ltr; font: small / 1.5 Arial, Helvetica, sans-serif; overflow: auto hidden; position: relative;"><div id="avWBGd-16"><div class="gmail_quote"><div dir="ltr"><div style="background-color: rgb(25, 26, 28); color: rgb(188, 190, 196);"><pre style="white-space: pre-wrap; font-family: &quot;JetBrains Mono&quot;, monospace; font-size: 9.8pt;"><span style="color: rgb(207, 142, 109);">import </span>pandas <span style="color: rgb(207, 142, 109);">as </span>pd<br><span style="color: rgb(207, 142, 109);">from </span>sklearn.model_selection <span style="color: rgb(207, 142, 109);">import </span>train_test_split<br><span style="color: rgb(207, 142, 109);">from </span>sklearn.metrics <span style="color: rgb(207, 142, 109);">import </span>accuracy_score<br><span style="color: rgb(207, 142, 109);">from </span>sklearn.linear_model <span style="color: rgb(207, 142, 109);">import </span>LogisticRegression<br>LR=LogisticRegression(<span style="color: rgb(170, 73, 38);">solver</span>=<span style="color: rgb(106, 171, 115);">'<wbr>liblinear'</span>)<br>data_df=pd.read_csv(<span style="color: rgb(106, 171, 115);">'heart.<wbr>csv'</span>)<br>x=data_df.drop(<span style="color: rgb(106, 171, 115);">'target'</span>,<span style="color: rgb(170, 73, 38);">axis</span>=<span style="color: rgb(42, 172, 184);">1</span><wbr>)<br>y=data_df[<span style="color: rgb(106, 171, 115);">'target'</span>]<br>x_train,x_test,y_train,y_test=<wbr>train_test_split(x,y,<span style="color: rgb(170, 73, 38);">test_size</span><wbr>=<span style="color: rgb(42, 172, 184);">0.2</span>,<span style="color: rgb(170, 73, 38);">random_state</span>=<span style="color: rgb(42, 172, 184);">42</span>)<br>LR.fit(x_train,y_train)<br><br>y_pred=LR.predict(x_test)<br>accuracy=accuracy_score(y_<wbr>test,y_pred)<br><span style="color: rgb(136, 136, 198);">print</span>(accuracy*<span style="color: rgb(42, 172, 184);">100</span>)<br>data_df = pd.read_csv(<span style="color: rgb(106, 171, 115);">'test_data.csv'</span>)<br>x = data_df.drop(<span style="color: rgb(106, 171, 115);">'target'</span>, <span style="color: rgb(170, 73, 38);">axis</span>=<span style="color: rgb(42, 172, 184);">1</span>)<br>y_pred = LR.predict(x)<br><span style="color: rgb(136, 136, 198);">print</span>(y_pred)<br><br></pre></div></div><div class="yj6qo"></div><div class="adL"></div></div><div class="adL"></div></div></div></div><div class="WhmR8e" data-hash="0" id="avWBGd-17" style="clear: both;"></div></div></div><div class="ajx" style="clear: both;"></div></div><div class="gA gt acV" style="font-size: 0.875rem; padding: 0px; width: auto; border-bottom-left-radius: 0px; border-bottom-right-radius: 0px; border-top: none; margin: 0px; background: rgb(255, 255, 255); color: rgb(34, 34, 34); font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="gB xu" jslog="184332; u014N:cOuCgd,Kr2w4b,xr6bB; 4:WyIjbXNnLWY6MTg1NTczNjU4NzI3MDkzNDM3MSIsbnVsbCxudWxsLG51bGwsMSwwLFsxLDAsMF0sOTksODMyLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCwxLG51bGwsbnVsbCxbMF0sbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsMCwwLG51bGwsbnVsbCxudWxsLG51bGwsWzEsMF1d" style="border-top: 0px; padding: 0px;"><div class="mVCoBd"></div><div class="ip iq" style="clear: both; margin: 0px; padding: 16px 0px; border-top: none;"><div class="pLw6bb"></div><div class="bJvOmf" id="avWBGd-18">
  |  
-- | --

</div></div></div></div>
</body>
</html>
